### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by raph-amiard**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+about: Ada language design problem that deserves a fix
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Problem
+
+One paragraph explanation of the problem, *with examples*
```
Comment (yakobowski) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]:
```suggestion
One-paragraph explanation of the problem, *with examples*
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+name: Bug report
+about: Ada language design problem that deserves a fix
```
Comment (yakobowski) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]: Remove design?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+about: Suggest a feature for the Ada language
+title: "[feature]"
+labels: enhancement
+assignees: ''
+
+---
+
+### Summary
+
+One paragraph explanation of the feature, *with examples of use*
```
Comment (yakobowski) [inline .github/ISSUE_TEMPLATE/feature_request.md:12]:
```suggestion
One-paragraph explanation of the feature, *with examples of use*
```

File: .github/ISSUE_TEMPLATE/language-feature.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+about: Suggest a feature for the Ada language
+title: "[feature]"
+labels: enhancement
+assignees: ''
+
+---
+
+### Summary
+
+One paragraph explanation of the feature, *with examples of use*
```
Comment (yakobowski) [inline .github/ISSUE_TEMPLATE/language-feature.md:12]:
```suggestion
One-paragraph explanation of the feature, *with examples of use*
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (raph-amiard) [reply .github/ISSUE_TEMPLATE/bug_report.md:3]: ok

Comment (yakobowski) [reply .github/ISSUE_TEMPLATE/bug_report.md:3]:
```suggestion
about: Ada language problem that deserves a fix
```

**Review by yakobowski — APPROVED** [review]

**Pull request merged**